en/docs/developer-guides/data-mapping/ai-data-mapping.md

```diff
@@ -0,0 +1,165 @@
+
+### 6. Every Mapping Is Validated
+
+Before the mapping is returned, it is checked by the **Ballerina Language Server** for:
+
+- Syntax correctness
+- Type safety
+- Required imports
+- Reserved keyword handling
+
+You always get code that compiles.
+
+## Real Healthcare Examples
+
+### Example 1: Custom Mapping (ClaimSubmission) to Standard Mapping([international401:Claim](https://central.ballerina.io/ballerinax/health.fhir.r4.international401/4.0.0#Claim))
+
+<a href="{{base_path}}/assets/img/developer-guides/data-mapping/ai-data-mapping/claim-submission-to-international401-claim.gif">
+<img src="{{base_path}}/assets/img/developer-guides/data-mapping/ai-data-mapping/claim-submission-to-international401-claim.gif" alt="AI Data Mapper transforming a custom ClaimSubmission record into a standard international401:Claim format" width="70%"></a>
+
+
+### Example 2: Standard Mapping ([hl7v23:PID](https://central.ballerina.io/ballerinax/health.hl7v23/4.0.1#PID)) to Standard Mapping ([international401:Patient](https://central.ballerina.io/ballerinax/health.fhir.r4.international401/4.0.0#Patient))
+
+<a href="{{base_path}}/assets/img/developer-guides/data-mapping/ai-data-mapping/hl7v23-pid-to-international401-patient.gif">
+<img src="{{base_path}}/assets/img/developer-guides/data-mapping/ai-data-mapping/hl7v23-pid-to-international401-patient.gif" alt="AI Data Mapper converting HL7V23 PID segment to international401:Patient format" width="70%"></a>
+
+## Responsible Use
+
+While **Large Language Models** have made remarkable progress, they can still produce unexpected results especially when dealing with highly domain-specific or unusual schema patterns. To ensure safe and accurate outcomes, we strongly recommend the following:
+
+- **Review all generated mappings** before using them in production.
+- **Test with representative and realistic data samples**.
+- **Validate that the generated logic aligns with your business rules**.
+- **Share feedback on incorrect or missing mappings** so the system can continue to improve.
+
+## The Future of Integration Development
+
+The **AI Data Mapper** marks a major shift in how integrations are built. Instead of spending hours manually matching fields, writing conversion logic, and tracking down type errors, developers can now offload the most repetitive and error-prone tasks to AI.
+
+This leaves you free to focus on what truly matters: building robust, scalable, business-critical systems.
+
+Whether you're:
+
+- Building new APIs
+- Orchestrating microservices
+- Constructing data pipelines
+- Implementing industry-specific interoperability solutions
+- Transforming between custom and standard data formats
+
+## Try the AI Data Mapper Today
+
+Upload your schemas even the big, scary ones and see how fast your largest integrations can move.
```

Comment:
Phrases like “big, scary ones” are informal and subjective

Lacks technical clarity (What makes a schema “big”? Size? Complexity?)

Sounds promotional rather than precise, which may reduce credibility in developer docs

Similar issues are there in most places. Check and fix.